Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -822,6 +820,19 @@ void ViewTransition::clearViewTransition()
         documentElement->invalidateStyleInternal();
 }
 
+LayoutRect ViewTransition::containingBlockRect()
```

Comment:
Can you add a link to https://drafts.csswg.org/css-view-transitions-1/#snapshot-containing-block so people know what this is referring to?